Core/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -74,10 +74,10 @@ private void AddNewlyReleasedVersionToClosedIssuesOnlyAssociatedWithFullVersion
       var allJiraVersionsStartingWithFullVersion = _jiraVersionCreator.FindAllVersionsStartingWithVersionNumber(currentFullVersion);
 
       if (allJiraVersionsStartingWithFullVersion.Count == 0)
-        throw new InvalidOperationException($"Could not find versions starting with '{currentFullVersion}'.");
+        throw new UserInteractionException($"Could not find versions starting with '{currentFullVersion}'.");
```

Comment:
```suggestion
        throw new UserInteractionException($"Could not find versions starting with '{currentFullVersion}' in JIRA.");
```

Depending on line 74, if this should just be pre-release versions, we should update this to `"No pre-release versions exist in JIRA for version '{currentFullVersion}'."`. It may be that this should be changed together with line 74 in a separate ticket.